web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -11,14 +11,105 @@ import Description from 'web-components/lib/components/description';
 import { Label } from 'web-components/lib/components/label';
 import { parseText } from 'web-components/lib/utils/textParser';
 import { formatDate, formatNumber } from 'web-components/lib/utils/format';
-import { getAccountUrl } from '../../lib/block-explorer';
-import { truncate } from '../../lib/wallet';
 
+import { getAccountUrl } from '../../lib/block-explorer';
 import { LinkWithArrow } from '../atoms/LinkWithArrow';
 import { OptimizedImage } from '../atoms/OptimizedImage';
 import topoImg from '../../assets/background-contour-2.svg';
 import forestImg from '../../assets/forest-token.png';
 
+type CreditClass = {
+  id: string;
+  name: string;
+};
+
+export type BasketOverviewProps = {
+  name: string;
+  displayDenom: string;
+  description: string;
+  totalAmount: number;
+  curator: string;
+  allowedCreditClasses: CreditClass[];
+  minStartDate?: string;
+  startDateWindow?: string;
+};
+
+export const BasketOverview: React.FC<BasketOverviewProps> = ({
+  name,
+  displayDenom,
+  description,
+  totalAmount,
+  curator,
+  allowedCreditClasses,
+  minStartDate,
+  startDateWindow,
+}) => {
+  const styles = useStyles();
+
+  const getDateCriteria = (
+    minStartDate?: string,
+    startDateWindow?: string,
+  ): string => {
+    if (minStartDate) return formatDate(minStartDate);
+    if (startDateWindow) return `${startDateWindow} rolling acceptance window`;
+    return '-';
+  };
+
+  return (
+    <SectionContainer>
+      <Section className={styles.content}>
+        <Grid container>
+          <ImageContainer item xs={12} sm={5}>
+            <OptimizedImage
+              className={styles.image}
+              src={forestImg}
+              alt={name}
+            />
+          </ImageContainer>
+          <TextContainer item xs={12} sm={7}>
+            <Title variant="h1" className={styles.title}>
+              {name}
+            </Title>
+            <Description className={styles.basketDenom}>
+              {displayDenom}
+            </Description>
+            <Description className={styles.basketDescription}>
+              {description}
+            </Description>
+            <OnBoardingCard className={styles.card}>
+              <Grid
+                container
+                rowSpacing={1}
+                columnSpacing={{ xs: 1, sm: 2, md: 3 }}
+              >
+                <Item label="total amount" data={formatNumber(totalAmount)} />
+                <Item
+                  label="curator"
+                  data={curator}
+                  link={getAccountUrl(curator as string)}
+                />
+                <ItemWithLinkList
+                  label="allowed credit classes"
+                  data={allowedCreditClasses}
+                  link={'/credit-classes/'}
+                />
+                <Item
+                  label="min start date"
```

Comment:
this label should be either "min start date" or "start date window"

Reply:
yes, it was not yet, I include it